capture/fswebcam.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+
+import capture
+from utils.log import debug
+
+
+class Fswebcam(capture.ImgCapture):
```

Comment:
(to be completely honest, I started with `FswebcamCaptureInterface`, an I found it waaaaaaay too long, even for a class name... But I'm willing to negociate if you agree to change a few identifiers in `imgprocessor` ;) )